src/constants.ts

```diff
@@ -77,3 +77,10 @@ export const ENDPOINTS = {
   geocoder:
     'https://api.data.amsterdam.nl/bag/nummeraanduiding/?format=json&locatie=',
 }
+
+/**
+ * Minimum and maximum possible zoom levels for maps
+ *
+ */
+export const MIN_ZOOM_LEVEL = 22
```

Comment:
just wondering, how are these zoom levels different from the zoomlevels defined in `DEFAULT_AMSTERDAM_MAPS_OPTIONS`?

Reply:
The DEFAULT_AMSTERDAM_MAPS_OPTIONS limits the capabilities of leaflet. These zoom levels provide the flexibility for use any other projects with any requirements.